### PR TITLE
Add Architecture tutorials

### PR DESCRIPTION
**Pull request by mamuz**
- Hexagonal Architecture
- The Onion Architecture

### CODE REVIEW

**Comment by mamuz** [comment]
First of all awesome collection of oop related articles :+1: 

I just add essential architecture related articles about layers, ports and adapters.

**Comment by jaymecd** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -189,6 +189,8 @@ A collection of resources to learn object-oriented programming and related conce
 * [The MVC Pattern and PHP, Part 1](http://www.sitepoint.com/the-mvc-pattern-and-php-1/) and [The MVC Pattern and PHP, Part 2](http://www.sitepoint.com/the-mvc-pattern-and-php-2/) - An introduction tutorial about MVC pattern.
 * [Alternatives To MVC](http://blog.ircmaxell.com/2014/11/alternatives-to-mvc.html) - An introduction to alternative architectures to MVC.
 * [N-Tier Architecture - An Introduction](http://blog.ircmaxell.com/2012/08/n-tier-architecture-introduction.html) - An introduction to n-tier architecture.
+* [Hexagonal Architecture](http://fideloper.com/hexagonal-architecture) - An introduction to also called Ports and Adapters architecture by Chris Fidao
```
Comment (Deveritas) [inline README.md]: The language is a little off at "An introduction to also called". Did you accidentally delete part of your sentence?

File: README.md [hunk 2]
```diff
@@ -189,6 +189,8 @@ A collection of resources to learn object-oriented programming and related conce
...
+* [The Onion Architecture : part 1](http://jeffreypalermo.com/blog/the-onion-architecture-part-1/) - An introduction to the Onion Architecture by Jeffrey Palermo
```
Comment (Deveritas) [inline README.md]: You're missing a period at the end of both of the lines as well.

**Comment by mamuz** [comment]
@Deveritas i changed what you mentioned.

**Comment by Deveritas** [comment]
(Y)

**Comment by Deveritas** [comment]
Sorry - github, not Skype.
I meant :+1:

**Comment by marcelgsantos** [comment]
Thank you so much for contributing and sorry for the long delay. ;)

**Pull request merged**